.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+
+  concurrency:
+    group: "release"
+    cancel-in-progress: false
+
+jobs:
+  publish-all:
+    runs-on: macos-latest
```

Comment:
I personally prefer to use a specific macOS version (same goes for Xcode when used).
Prevents me from staring at strange errors when Github decides to update their `latest` tag 😇.